department-of-veterans-affairs/abd-vro
# Add BIP CLAIM ISS to HELM

## What was the problem?
<strike>Bip claim ISS was not available through helm, and was relying on application.yml defaults</strike>
BIP_CLAIM_ISS should not be an environment variable as it never really needs to change.
The same is true of BIP_EVIDENCE_ISS
Other cleanup was found while removing the first.
Associated tickets or Slack threads:
https://amida.atlassian.net/browse/MCP-2366
- [MCP-2366](https://amida.atlassian.net/browse/MCP-2366)

## How does this fix it?
Unused BIP_APPLICATION_ISS is removed
Unused JTI in BIP Api Props is removed
<strike>For Non local deployments- Helm now has BIP_CLAIM_ISS that should flow through the yml files.</strike>
<strike>For local deployments- setenv sets the variable, and it goes to the yml files that way. </strike>
BIP_CLAIM_ISS and BIP_EVIDENCE_ISS now just run from the yml with no environment variables present. 

## How to test this PR
Locally i can verify that everything still stands up and runs. <strike>@au2 if there is something else I can do to test helm, let me know. Or if this is not the approach you wanted for local, I saw the note in setenv that you tried to move them to the yml but that seems to be the opposite of what's desired here?</strike> these variables now do not change.
